data/sriov/002-sriov-cni.yaml

```diff
@@ -31,6 +31,14 @@ spec:
       - name: kube-sriov-cni
         image: {{ .SriovCniImage }}
         imagePullPolicy: {{ .ImagePullPolicy }}
+        command:
+          - /bin/bash
+          - -c
+          - |
+            cp -f /usr/src/sriov-cni/bin/*sriov /host/opt/cni/bin/
+            cp -f /usr/bin/*sriov /host/opt/cni/bin/
```

Comment:
@booxter can you explain why you add two copies?

Reply:
Described in PR/commit message. It is just so we are compatible with both old and new SR-IOV image releases (they keep binaries in different dir).

Reply:
miss that thanks!